pkg/controller/servicebinding/sbrcontroller.go

```diff
@@ -145,6 +145,7 @@ func buildGVKPredicate(logger *log.Log) predicate.Funcs {
 		UpdateFunc: updateFunc(logger),
 		DeleteFunc: func(e event.DeleteEvent) bool {
 			// evaluates to false if the object has been confirmed deleted
+			logger.Debug("Resource update event received", "GVK", e.Object.GetObjectKind(), "name", e.Meta.GetName())
```

Comment:
do we really need this change in PR? BTW, the message is misleading, since we receive here delete event.